packages/metalk8s-sosreport.spec

```diff
@@ -0,0 +1,38 @@
+Name:		metalk8s-sosreport
+Version:	1.0.0
+Release:	1%{?dist}
+Summary: 	Metalk8s SOS report custom plugins
+
+
+Requires: python >= 2.6, python < 2.8
```

Comment:
Is the `python < 2.8` really needed? I don't think Python 2.8 will ever exists. Or maybe simple `python = 2.7`?

Reply:
we want all versions of `2.7.X` so we go for ` < 2.8`